Documentation/architecture.rst

```diff
@@ -255,6 +255,7 @@ Load Balancer            node             64k             Max 64k cumulative bac
 Policy                   endpoint         16k             Max 16k allowed identity + port + protocol pairs for specific endpoint
 Proxy Map                node             512k            Max 512k concurrent redirected TCP connections to proxy
 Tunnel                   node             64k             Max 32k nodes (IPv4+IPv6) or 64k nodes (IPv4 or IPv6) across all clusters
+IPv4 Fragmentation       node             8k              Max 8k fragmented datagrams in flight on node
```

Comment:
nit: `Max 8k fragmented datagrams in flight simultaneously on the node`?

Reply:
Should we make this size configurable? So a user can configure a table of more than 8k?

Reply:
Will address the nit. For the configurable size, I don't know. We don't expect many IPv4 fragments in the first place, so the reasoning was that 8k should be probably be enough. Should we wait and see if there is demand for more entries, or would you prefer me to add an option now?

Reply:
I presume in normal case it might be only DNS related or not used at all, unless a user misconfigures their network. Maybe in that sense 1k would even be enough as a default as we prealloc the whole map? Given we have tunables for `bpf-policy-map-max` for example, it would probably make sense to add one here if we have it on a lower size by default.

Reply:
Ok, I labelled the follow-up PR as WIP, will add a commit to introduce a tunable here as well.